docs/dasharo-menu-docs/dasharo-system-features.md

```diff
@@ -19,6 +19,18 @@ This menu offers security-sensitive options like:
   BIOS flash. Recommended to be enabled if [Firmware setup password](overview.md#user-password-management)
   is set. Disable the protection if you need access to flash with flashrom.
 
+- `Dma protection` - Early Boot DMA protection is a feature that utilizes the
+  IOMMU early in the boot process to prevent rogue DMA-capable devices such as
+  PCIe add-in cards and USB 3 devices from access to memory. This prevents a
+  class of DMA attacks that allow for exfiltration of secrets and installation
+  of malware early in the boot process.
+- `IOMMU handoff at ExitBootServices` - this option is available when
+  `Dma protection` is enabled. It changes the moment when IOMMU is re-enabled.
+  If this option is selected, then it's re-enabled at ExitBootServices when
```

Comment:
The thing is that the IOMMu is being disabled when handing-off to OS to not cause problems with PCI enumeration. OS should set up IOMMU per its own needs during the handoff state. The option here is to mitigate problems with OSes that do not work well with EBS handoff.